Source/hydro/Castro_hydro.cpp

```diff
@@ -129,6 +129,12 @@ Castro::cons_to_prim_fourth(const Real time)
 
     MultiFab& S_new = get_new_data(State_Type);
 
+#ifdef MHD
+    MultiFab& Bx = get_new_data(Mag_Type_x);
+    MultiFab& By = get_new_data(Mag_Type_y);
+    MultiFab& Bz = get_new_data(Mag_Type_z);
+#endif
+
```

Comment:
are these needed in Castro_hydro.cpp?

Reply:
nope, doesn't seem like we need them there